### DISCUSSION TITLE
Welcome to paper-qa Discussions!

### DISCUSSION THREAD

**Discussion by whitead** [Future-House/paper-qa#76] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stonebyun** [comment]
Glad to meet you all!
My name is Kang. I am interested in self-organizing knowledge model or knowledge FSM.

**Comment by andreifoldes** [comment]
Greetings,

Much thanks for the tremendous work on paper-qa. I was wondering whether given the recent announcement regarding the GPT4-Turbo and the increased context window, whether there is any outlook on how to incorporate that model into the recommended paperqa workflow?

**Comment by Snikch63200** [comment]
Hello,

Thank you for your amazing work ! I am currently working on a massive bibliographic research tool for archeology and PaperQA gives very encouraging results. I guess you are in an very active developpement phasis but I wonder if you planned to provide a complete API documentation in the future ?

Best regards.

**Comment by lavericklavericklaverick** [comment]
Hi Andrew, great project, thanks for all the hard work.

Technical question: how do you go about searching for parameters with scientific notation? From my experiments in OAI assistants it appears to be impossible to embed them correctly as they are not recognised tokens. For example say if i wanted to ask 'What is \sigma_{\Delta K}?' it could give me an approximate result but not exact (mostly through pot luck of dumping in a load of chunks).

This appears to be a challening aspect of a paper search tool for me, it needs to be general enough to allow for small changes in pdf formatting / parsing but also very exact as not to give the wrong values.

I would love to hear your/ everyones thoughts on this. Thanks

**Comment by foreverallama** [comment]
Hi! 
Thanks for the project! Great that its open-source as well, I had the opportunity to contribute to the JSON parser at some point and it enabled a lot of learning!

Just curious, are you guys planning on submitting something over to `Open Conference of AI Agents for Science 2025`? Would love to hear about the latest you're working on!

**Reply by jamesbraza** [reply]
Hi @foreverallama yes thanks for the contributions please keep them coming.

> Just curious, are you guys planning on submitting something over to `Open Conference of AI Agents for Science 2025`? Would love to hear about the latest you're working on!

We are not submitting anything there, actually only I recently learned about this one. Are you submitting anything there?

**Reply by foreverallama** [reply]
No, even I learned about it a bit too late 😅. I still tried a bit; started with some literature review with PaperQA+Gemini and thought it was quite capable, though I was hitting rate limits quick when using larger top-K evidence retrievals. Been trying out Perplexity as well, I find its quite good when working with a small number of documents. 

Anyways, curious to see how this project evolves. Will try to contribute wherever I can!
